src/core_plugins/kibana/public/dashboard/partials/save_dashboard.html

```diff
@@ -11,12 +11,18 @@
     placeholder="Dashboard title"
     input-focus="select"
   >
+
+  <saved-object-save-as-check-box saved-object="opts.dashboard"></saved-object-save-as-check-box>
```

Comment:
Ah, nice catch!  I opted to throw the variable into savedObject so I wouldn't have to add that input parameter to every user of the check box.